src/popup/popup_style.css

```diff
@@ -0,0 +1,55 @@
+.grid-container-form {
```

Comment:
I suggest 2 spaces indent for css files. This would be consistent with our existing css file in `src/templates`. This file contains a mix of 4 spaces and tabs..